build.sbt

```diff
@@ -128,6 +128,7 @@ lazy val `support-frontend` = (project in file("support-frontend"))
     buildInfoKeys := BuildInfoSettings.buildInfoKeys,
     buildInfoPackage := "app",
     buildInfoOptions += BuildInfoOption.ToMap,
+    scalacOptions += "-Ytasty-reader",
```

Comment:
needed to tell scala 2.13 that it is allowed to read scala 3 `.tasty` files instead of scala 2.13 "pickles" alongside the `.class` files to understand the code it's compiling against.

Reply:
ah, that makes sense - we're saying that the main `support-frontend` project will stay on Scala 2.13 for now, but by adding the [`-Ytasty-reader`](https://docs.scala-lang.org/scala3/guides/migration/compatibility-classpath.html#a-scala-213-module-can-depend-on-a-scala-3-artifact) flag, it's allowed to depend on the `support-internationalisation` module, which is using Scala 3.